feature-libs/asm/root/model/customer-lists-page.model.ts

```diff
@@ -0,0 +1,14 @@
+import { UserGroup } from './user-group.model';
```

Comment:
can we put these model interfaces into one file something like this: `customer-list.model.ts`